cluster-install/install.sh

```diff
@@ -2376,7 +2638,21 @@ step_generate_torc_patch() {
         torc_patch_list+='{"op":"replace","path":"/spec/autosupportImage","value":"'"$autosupport_image"'"},'
     fi
 
-    if [ "${#torc_patch_list[@]}" -gt 0 ]; then
+    # Update image pull secrets if needed
+    if [ -n "$IMAGE_PULL_SECRET" ]; then
+        if echo "$_EXISTING_TORC_PULL_SECRETS" | grep -q "^${IMAGE_PULL_SECRET}$" &> /dev/null; then
+            logdebug "image pull secret '$IMAGE_PULL_SECRET' already present in torc"
+        else
+            local -r secret_obj='{"name": "'"$IMAGE_PULL_SECRET"'"}'
+            if [ -z "$_EXISTING_TRIDENT_OPERATOR_PULL_SECRETS" ]; then
+                torc_patch_list+='{"op":"replace","path":"/spec/imagePullSecrets","value":"'"$IMAGE_PULL_SECRET"'"},'
+            else
+                torc_patch_list+='{"op":"add","path":"/spec/imagePullSecrets/-","value":"'"$IMAGE_PULL_SECRET"'"},'
+            fi
+        fi
+    fi
```

Comment:
Same here, but for the TORC -- generate a patch to add the IMAGE_PULL_SECRET.

Reply:
Same comment about the values. I'll make the change on your behalf :D